packages/plugin-docusaurus-v3/package.json

```diff
@@ -16,23 +16,33 @@
     "url": "git+https://github.com/orama/orama.git"
   },
   "sideEffects": false,
-  "main": "src/server/index.js",
-  "scripts": {},
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "build": "tsc",
+    "postbuild": "sh scripts/postbuild.sh",
+    "watch": "tsc --watch"
+  },
   "dependencies": {
     "@docusaurus/plugin-content-docs": "^3.0.1",
     "@docusaurus/theme-common": "^3.0.1",
     "@orama/highlight": "^0.1.2",
-    "@orama/orama": "workspace:*",
```

Comment:
@raiindev you probably want to have `workspace:*` back here and in the line below